components/FeatureHighlight/src/MDCFeatureHighlightViewController.m

```diff
@@ -48,15 +51,48 @@ - (nonnull instancetype)initWithHighlightedView:(nonnull UIView *)highlightedVie
     _completion = completion;
     _animationController = [[MDCFeatureHighlightAnimationController alloc] init];
     _animationController.presenting = YES;
+    _outerHighlightColor =
+        [[UIColor blueColor] colorWithAlphaComponent:kMDCFeatureHighlightOuterHighlightAlpha];
+    _innerHighlightColor = [UIColor whiteColor];
 
     _displayedView.accessibilityTraits = UIAccessibilityTraitButton;
 
+    _viewAccessiblityHint = [[self class] dismissAccessibilityHint];
+
     super.transitioningDelegate = self;
     super.modalPresentationStyle = UIModalPresentationCustom;
   }
   return self;
 }
 
+- (void)viewDidLoad {
+  [super viewDidLoad];
+
+  self.featureHighlightView.displayedView = _displayedView;
+  self.featureHighlightView.autoresizingMask =
+      UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
```

Comment:
This would expect autoresizingMask to be explicitly set / handled in featureHighlightView.
A view controller should only be changing the state of it's child views that it needs to, typically frame.  (And in our case properties that we need to push through, adjustsFont..., colors, fonts)

Reply:
Oh my. Is `featureHighlightView` not a subview?

Reply:
*facepalm*

Ok. If self.view == self.featureHighlightView then the above statement still holds true, and we definitely should not be setting any of those properties here.

Reply:
Indeed, self.view == self.featureHighlightView
I think we are in agreement that frame / bounds, center and transform typically include the UIView properties that a view controller is 'allowed' to adjust on its subviews.
I would expect a view to handle it's autoresizingMask and autoresizesSubviews internally.

Reply:
autoresizingMask is a parent responsibility, autoresizesSubviews is the view's responsibility.

Like .frame (center + bounds + transform), a view lacks the contextual awareness to be able to set an autoresizing mask. A view that enforces its own layout behavior via setting its own autoresizingMask will very quickly break layout in the parent view.

So the concrete requirement for this PR as-is is to remove the autoresizingMask modification because this view controller lacks the parent context required to know if a flexible width/height is truly desired (e.g. maybe the feature highlight view is being shown in a subset of the screen and should have a flexible top/left margin instead — it's the responsibility of the owner of this feature highlight view to decide that).

Reply:
This seems like something we completely need to fix but I actually disagree with making this change here since this PR is unrelated to this topic or issue which already existed in this VC. We can create a separate issue to address that and fix it. I don't have as much context as you two @jverkoey @ianegordon  so would you guys mind creating a github issue for that.

Reply:
Agreed @mohammadcazig! Filing now.

Reply:
https://github.com/material-components/material-components-ios/issues/4262

Reply:
I'm not sure I follow on autoresizingMask.  Why don't we catch up in person tomorrow, with a whiteboard / visual aids?  We are currently not consistent across our components, and we definitely should be.  Thanks for opening the issue.